redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1212 @@
+            mutes_data = self._server_mutes[ctx.guild.id]
+            to_rem = []
+            if mutes_data:
+
+                msg += _("__Server Mutes__\n")
+                for user_id, mutes in mutes_data.items():
+                    if not mutes:
+                        to_rem.append(user_id)
+                        continue
+                    user = ctx.guild.get_member(user_id)
+                    if not user:
+                        user_str = f"<@!{user_id}>"
+                    else:
+                        user_str = user.mention
+                    if mutes["until"]:
+                        time_left = timedelta(
+                            seconds=mutes["until"] - datetime.now(timezone.utc).timestamp()
+                        )
+                        time_str = humanize_timedelta(timedelta=time_left)
+                    else:
+                        time_str = ""
+                    msg += _("{member}").format(member=user_str)
+                    if time_str:
+                        msg += _("__Remaining__: {time_left}\n").format(time_left=time_str)
+                    else:
+                        msg += "\n"
+            for _id in to_rem:
+                try:
+                    del self._server_mutes[ctx.guild.id][_id]
+                except KeyError:
+                    pass
+        for channel_id, mutes_data in self._channel_mutes.items():
+            if not mutes_data:
+                to_del.append(channel_id)
+                continue
+            if channel_id in [c.id for c in ctx.guild.channels]:
+                msg += _("__<#{channel_id}> Mutes__\n").format(channel_id=channel_id)
+                for user_id, mutes in mutes_data.items():
+                    if not mutes:
+                        continue
+                    user = ctx.guild.get_member(user_id)
+                    if not user:
+                        user_str = f"<@!{user_id}>"
+                    else:
+                        user_str = user.mention
+                    if mutes["until"]:
+                        time_left = timedelta(
+                            seconds=mutes["until"] - datetime.now(timezone.utc).timestamp()
+                        )
+                        time_str = humanize_timedelta(timedelta=time_left)
+                    else:
+                        time_str = ""
+                    msg += _("{member} ").format(member=user_str)
+                    if time_str:
+                        msg += _("__Remaining__: {time_left}\n").format(time_left=time_str)
+                    else:
+                        msg += "\n"
+        for c in to_del:
+            del self._channel_mutes[c]
+        if msg:
+            for page in pagify(msg):
+                await ctx.maybe_send_embed(page)
+            return
+        await ctx.maybe_send_embed(_("There are no mutes on this server right now."))
+
+    @commands.command(usage="[users...] [time_and_reason]")
```

Comment:
Users argument is not optional.
```suggestion
    @commands.command(usage="<users...> [time_and_reason]")
```